docs/ui-damage-overlay.md

```diff
@@ -0,0 +1,14 @@
+# Damage Overlay
+
+The `Health` module adds a visual overlay whenever the you are damaged.
+If you hurt yourself, for instance through drowning or fall damage, the default damage overlay is displayed.
+It consists of four red indicators, one on each side of your screen.
+The default damage overlay is also shown when you are hurt by an effect such as poison.
```

Comment:
This might again warrant a bit more distinctiveness (or a strive toward that at some point in the future)

* Player X died from fall damage after jumping off a cliff!
* Player X died from fall damage after being pushed off a cliff by <instigator>
* Player X died from poisoning after eating some bad puffer fish!
* Player X died from poisoning after being hit by a grave-seeker arrow shot by <instigator>!

Actually, those examples are less about hurting oneself as being about whether there is another instigator or not 🤔

Reply:
see my comment above

Reply:
Maybe like this? We directly pick up the same wording as used in another place already.

> Self-inflicted causes include, for instance, falling, drowning, or poison damage.
> Damage inflicted by a third party typically involves some form of hitting the affected entity.

```suggestion
If you hurt yourself (self-inflicted damage), for instance through drownin, fall damage, or poison, the default damage overlay is displayed.
It consists of four red indicators, one on each side of your screen.
```